FreewheelSampleApp/app/src/main/java/com/ooyala/sample/players/PreconfiguredFreewheelPlayerActivity.java

```diff
@@ -24,14 +28,17 @@
  * In order for Freewheel to work this simply, you need the following parameters set in your Third Party Module Parameters
  * - fw_android_ad_server
  * - fw_android_player_profile
- * 
+ *
  * And an Freewheel Ad Spot configured in Backlot with at least the following:
  * - Network ID
  * - Video Asset Network ID
  * - Site Section ID
- * 
+ *
  */
 public class PreconfiguredFreewheelPlayerActivity extends Activity implements Observer {
+  private static final int MY_PERMISSIONS_REQUEST_READ_CONTACTS = 0;
+  private String text;
```

Comment:
instead of adding this text variable here, do something else. For example, use a boolean flag that will be `true` when you get the permission to `WRITE_EXTERNAL_STORAGE`